packages/twenty-front/src/modules/navigation/components/MainNavigationDrawerItems.tsx

```diff
@@ -15,7 +18,42 @@ import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
 
 import { useIsTasksPage } from '../hooks/useIsTasksPage';
 
-export const MainNavigationDrawerItems = () => {
+const StyledSkeletonContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 6px;
+  margin-left: 12px;
+`;
+
+const StyledSkeletonLoader = ({
```

Comment:
this is not a Styled component (it's actually containing much more) so we should not prefix it with Styled and it should go in its own file NavigationDrawerItemsSkeletonLoader for example